### PR TITLE
docs(plugin/openwhisk) fix grammar and config description

### PR DESCRIPTION
**Pull request by shashiranjan84**
- Updated `timeout` description 
- Updated `keepalive` description
- Fixed some typo

### CODE REVIEW

File: app/plugins/openwhisk.md [hunk 1]
```diff
@@ -76,9 +76,10 @@ Attribute                                | Description
 `config.service_token`<br>*optional*     | The service token to access Openwhisk resources.
 `config.https_verify`<br>*optional*      | Set it to true to authenticate Openwhisk server. Defaults to `false`.
 `config.https`<br>*optional*             | Use of HTTPS to connect with the OpenWhisk server. Defaults to `true`.
-`config.timeout`<br>*optional*           | Timeout in seconds before aborting a connection to OpenWhisk server. Defaults to `60 seconds`.
-`config.result`<br>*optional*            | Return only the result of the Action invoked. Defaults to `true`.
-`config.keepalive`<br>*optional*         | Defines for how long an idle connection to OpenWhisk server will live before being closed. Defaults to `60 seconds`.
+`config.timeout`<br>*optional*           | Timeout in millisecond before aborting a connection to OpenWhisk server. Defaults to `60000`.
```
Comment (theetrain) [inline app/plugins/openwhisk.md]: > Timeout in `milliseconds` before aborting `connection` to OpenWhisk server.

File: app/plugins/openwhisk.md [hunk 2]
```diff
@@ -76,9 +76,10 @@ Attribute                                | Description
...
+`config.result`<br>*optional*            | Return only the result of the `Action` invoked. Defaults to `true`.
+`config.keepalive`<br>*optional*         | Time in millisecond for which an idle connection to OpenWhisk server will live before being closed. Defaults to `60000`.
```
Comment (theetrain) [inline app/plugins/openwhisk.md]: > Time in `milliseconds` ...

File: app/plugins/openwhisk.md (hunk 1 above)
Comment (thibaultcha) [inline app/plugins/openwhisk.md]: We don't use back ticks around units of time. I don't see why we would use them around connection either...

Comment (thibaultcha) [inline app/plugins/openwhisk.md]: Or maybe you wanted to highlight that you found a typo in the word (plural form?), in which case, sorry!

**Comment by shashiranjan84** [comment]
@theetrain made it plural.

**Pull request merged**